src/openforms/formio/components/vanilla.py

```diff
@@ -62,6 +67,39 @@ class Default(BasePlugin):
 class TextField(BasePlugin[TextFieldComponent]):
     formatter = TextFieldFormatter
 
+    def build_serializer_field(
+        self, component: TextFieldComponent
+    ) -> serializers.CharField | serializers.ListField:
+        multiple = component.get("multiple", False)
+        validate = component.get("validate", {})
+        required = validate.get("required", False)
+
+        if validate.get("plugins", []):
+            raise NotImplementedError("Plugin validators not supported yet.")
```

Comment:
Are we certain that no components currently have plugins set? If not, maybe we should just ignore them.

If this is purely a formio thing, then it's probably fine

Reply:
no, this will break and I will resolve it in the next few days

this is a negative side-effect of ignoring coverage for `NotImplementedError` btw, could've ended up in runtime like this without being aware :/